doc/en/configuration_guide/imp/index.rst

```diff
@@ -12,9 +12,10 @@ working condition your Centreon platform. Templates of monitoring ready to use i
 procedure that allow to start your monitoring less than 30 minutes after the installation of your
 Centreon platform.
 
-Centreon Plugins' project offer a large range of functionality and is one of the richest in the
-market: +170 environments is around 2000 monitoring indicators which allow to deploy your monitoring
-easily and quickly.
+The Plugin Packs (configuration pack), developed by Centreon, are based on the
+Centreon Plugins (monitoring probes), offer a large range of functionality and
+is one of the richest in the market: +170 environments is around 2000 monitoring
```

Comment:
+170 environments is around 2000 monitoring -> +170 environments for almost 2000 monitoring

Reply:
corrected